custom-nginx-reverse-proxy.conf

```diff
@@ -1,14 +1,26 @@
-worker_processes 1;
-
-events { worker_connections 1024; }
+worker_processes auto;
+
+events {
+    accept_mutex on;
+    multi_accept on;
+    use epoll;
```

Comment:
I've had a lot of issues with docker + nginx, so I run nginx locally using this config.

> On platforms that support several methods nginx will normally select the most efficient method automatically

According to https://nginx.org/en/docs/events.html#kqueue it seems that `use kqueue` would be better for my local env, I wonder if `epoll` would just be selected by default when running in the docker (linux) container anyway.

Reply:
Kqueue is best for BSD. Nginx defaults to most efficient method: http://nginx.org/en/docs/ngx_core_module.html#use
It's better to have it (the default) documented in the config.

Reply:
> Nginx defaults to most efficient method

Right, so it'll now pick epoll for linux and epoll for MacOS, but without this it would *still* pick epoll for linux, and kqueue for MacOS. i.e. can we remove this and let it pick the default